charts/komodor-agent/values.yaml

```diff
@@ -54,6 +54,8 @@ capabilities:
   metrics: true
   # capabilities.networkMapper -- (bool) Enable network mapping capabilities by the komodor agent
   networkMapper: true
+  # capabilities.nodeEnricher -- (bool) Enable node enricher capabilities by the komodor agent
+  nodeEnricher: true
```

Comment:
I'd start by disabling it by default, testing on our own clusters for a few days and only then enabling it for customers. We do have the new agent release process which should validate it enough before the actual release but sync with @Moran-k to make sure how it works.